### DISCUSSION TITLE
Issue: Web man blackscreen at loading game from folder at HFW 4.91

### DISCUSSION THREAD

**Discussion by boludoz** [aldostools/webMAN-MOD#1088] [Q&A]
The problem at hand is that my screen goes black when I try to mount a game from the folder with Webman. The games mount popperly but, when i start that it stay at balckscreen.

It happens to several people and I'm using super slim PS3. And webman at latest release.

**Comment by aldostools** [comment]
Blackscreen issue can be due by different causes:
- Dynamic fan control (normally fixed setting SYSCON or manual fan mode)
- Games in folder format
- EBOOT signed improperly for HEN
- Game controller connected to USB
- Other causes

**Reply by boludoz** [reply]
So Maybe that is my controller and dynamic fan control

**Comment by boludoz** [comment]
The BLES version of UFC 3 has worked for me, even with USB control and fan in dynamic mode.

**Reply by boludoz** [reply]
It's just a curious fact.

**Reply by aldostools** [reply]
The dynamic fan causes the black screen only when there is a change of temperature during the game launch. The dynamic fan detects the change and makes changes in memory that cause a kernel panic by the hypervisor.

Some games don't trigger the dynamic fan when they are launched.

**Reply by boludoz** [reply]
How interesting, thanks for the explanation and for the good work.
